<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Measurement</class>
 <widget class="QDialog" name="Measurement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New measurement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="bullButton">
       <property name="text">
        <string>Bull</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cowButton">
       <property name="text">
        <string>Cow</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="AgeLabel">
       <property name="text">
        <string>Age(years)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="AgeTxt"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Height</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="HWithersLabel">
        <property name="text">
         <string>At the withers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="HWithersTxt"/>
      </item>
      <item>
       <widget class="QLabel" name="HSacrumLabel">
        <property name="text">
         <string>In the sacrum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="HSacrumTxt"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="DBreastLabel">
       <property name="text">
        <string>Breast depth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="DBreastTxt"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Width</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="WBreastLabel">
        <property name="text">
         <string>Breast behind blades</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="WBreastTxt"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="WHookLabel">
        <property name="text">
         <string>Hook bones</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="WHookTxt"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Label">
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="WTuberosityTxt"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Cross-length</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="CrTrunkLabel">
        <property name="text">
         <string>Trunk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="CrTrunkTxt"/>
      </item>
      <item>
       <widget class="QLabel" name="CrSeatLabel">
        <property name="text">
         <string>Seat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="CrSeatTxt"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Girth</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="GBreastLabel">
        <property name="text">
         <string>Breast</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="GBreastTxt"/>
      </item>
      <item>
       <widget class="QLabel" name="GMetacarpusLabel">
        <property name="text">
         <string>Metacarpus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="GMetacarpusTxt"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="HalfSeatLabel">
       <property name="text">
        <string>Half-girth of seat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HalfSeatTxt"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
